src/azure-cli/azure/cli/command_modules/profile/custom.py

```diff
@@ -30,6 +31,20 @@
                                 " Run 'az login' only if you need to use a different account")
 
 
+LOGIN_ANNOUNCEMENT = (
+    "[Announcements]\n"
+    "With the new Azure CLI login experience, you can select the subscription you want to use more easily. "
+    "Learn more about it and its configuration at https://go.microsoft.com/fwlink/?linkid=2271236\n\n"
+    "If you encounter any problem, please open an issue at https://aka.ms/azclibug\n"
+)
+
+LOGIN_OUTPUT_WARNING = (
+    "[WARNING] The login output has been updated. Please be aware that it no longer displays the full list of "
```

Comment:
To align with `[Announcements]`:

```suggestion
    "[Warning] The login output has been updated. Please be aware that it no longer displays the full list of "
```